examples/experimental/js/with-mapbox-map/app.js

```diff
@@ -67,15 +67,16 @@ class App {
   }
 
   onViewportChange(viewport) {
+    Object.assign(viewport, {viewState: viewport});
```

Comment:
We should never modify objects that are passed to us. That can surprise the caller.

Suggest
```
this.setProps({viewState: viewport});
```
or if we also need the viewport params on top level
```
this.setProps(Object.assign({}, viewport, {viewState: viewport}));
```